utils/transaction-parser.ts

```diff
@@ -342,9 +376,18 @@ export function parseZetaTransaction(
       }
     }
   );
+
+  let parsedEvents = [];
+  eventParser.parseLogs(tx.meta.logMessages, (event) => {
```

Comment:
Would this work or no?
```suggestion
 parsedEvents = tx.meta.logMessages.map(eventParser.parseLogs);
```

Reply:
I can't get this to run without compiler errors 😅 My syntax is what we do over in the SDK too: https://github.com/zetamarkets/sdk/blob/main/src/utils.ts#L613-L616

Reply:
Ok cool roll with that then, just thought functional might be cleaner but idk syntax off top of my head